dashboard/lib/tasks/seed.rake

```diff
@@ -160,10 +160,6 @@ namespace :seed do
     end
   end
 
-  task import_custom_levels: :environment do
-    LevelLoader.load_custom_levels
-  end
```

Comment:
This task is identical to the `:custom_levels` task below, and does not seem to be referenced anywhere.